src/app/organisations/index.js

```diff
@@ -63,6 +64,7 @@ const routes = () => {
   router.post("/v3/users", asyncWrapper(listUserOrganisationsV3));
   router.get("/invitations", asyncWrapper(listInvitationOrganisations));
   router.post("/", asyncWrapper(createOrganisation));
+  router.patch("/:id/edit", asyncWrapper(editOrganisation));
```

Comment:
Sorry, I missed this when I first reviewed it.  By nature of it being PATCH, you don't need have the `/edit` at the end of this endpoint, because the verb already describes what it's doing to the resource. Can you update the endpoint to just be `/:id`